### ISSUE TITLE
Rerooting does not place branch labels correctly at the root

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rambaut/figtree#10]
```
From Heiko Schmidt:

I just encountered one possible error which is obvious to
me but might not be to other users. If you use the node label
as the place to store bootstrap or other support values as done
by some programs (like MrBayes, clustalw, TREE-PUZZLE...),
you can use this as branch label in FigTree.
However, if you now reroot the tree the label is then moved
to the branch now root-wards of the respective node, which is
not the branch anymore corresponding to the support value.

Hence, my question could you maybe add a branch-label-class
(like "label, support") that can deal with this problem?
Off course there might be lables where this behaviour might
be the wanted one.


```

Original issue reported on code.google.com by `ramb...@gmail.com` on 13 Feb 2008 at 9:05

**Comment by AlesBucek** [comment]
Was this really resolved? I noticed (in FigTree 1.4.4) that rerooting a tree with bootstrap supports results in some incorrect labeling of nodes/branches around the root - the labels seems to be shifted. I will be glad to share an example tree where I experienced this.

**Comment by kks-vir** [comment]
@AlesBucek
I noticed the same, did you find any solution for it?
